data/data-pipeline/data_pipeline/application.py

```diff
@@ -121,5 +122,55 @@ def generate_map_tiles():
     generate_tiles(data_path)
 
 
+@cli.command(
+    help="Data Full Run (Census download, ETLs, score, combine and tile generation)",
+)
+@click.option(
+    "-c",
+    "--check",
+    is_flag=True,
+    help="Check if data run has been run before, and don't run it if so.",
+)
+def data_full_run(check):
+    """CLI command to run ETL, score, JSON combine and generate tiles in one command
+
+    Args:
+        check (bool): Run the full data run only if the first run sempahore file is not set (optional)
+
+     Returns:
+        None
+    """
+    data_path = settings.APP_ROOT / "data"
+
+    if check and not check_first_run():
+        # check if the data full run has been run before
+        logger.info("*** The data full run was already executed")
+        exit()
+
+    # census directories
+    logger.info("*** Initializing all data folders")
+    census_reset(data_path)
+    data_folder_cleanup()
+    score_folder_cleanup()
+    temp_folder_cleanup()
+
+    logger.info("*** Downloading census data")
+    etl_runner("census")
+
+    logger.info("*** Running all ETLs")
+    etl_runner()
```

Comment:
does this re-run census since it is supposed to run all ETLs, or does it skip census?

Reply:
It executes the Census ETL, yes. The Census ETL does have some caching logic to not re-download files that already exist in the file system but it does go through the SHP files extraction regardless.